#
#
#
#
#

Header header

int32 refId
int32 loopClosureId

uint32 actuatorStep
float32[] actuators